artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -1679,7 +1699,8 @@ public int getEncodeSize() {
          BufferHelper.sizeOfNullableInteger(idCacheSize) +
          BufferHelper.sizeOfNullableSimpleString(pageFullMessagePolicy != null ? pageFullMessagePolicy.toString() : null) +
          BufferHelper.sizeOfNullableInteger(prefetchPageBytes) +
-         BufferHelper.sizeOfNullableInteger(prefetchPageMessages);
+         BufferHelper.sizeOfNullableInteger(prefetchPageMessages) +
+         BufferHelper.sizeOfNullableBoolean(autoCreateDivertDestination);
```

Comment:
I'm not sure this is needed any longer... we are not changing the wires for the old methods.

Preetty much just use JSON for anything new now.. older stuff should stay as is.